subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -4605,6 +4610,15 @@ static inline void event_phy_upd_ind_prep(struct ll_conn *conn,
 			conn->llcp_ack = conn->llcp_req;
 		}
 
+		/* supported PHYs mask */
```

Comment:
wondering why we should wait until instant to check this? Why not do it when we check the _IND?

Reply:
There are two allowed pass criteria:
1. To disconnect if peer uses unsupported PHY
2. Continue the connection on old PHY

For backwards compatibility and not to change the controller behavior, option 2 is selected. 
Under option 2, the new PHY is committed at the instant, and this is where the decision to not have a PHY update event towards Host is implemented, hence I added the checks here.

Reply:
OK, I get that, and I also now understand that in case only one of the PHYs are suggesting an unsupported PHY we need to process the other one, so we have to handle this at instant.

Reply:
And this needs to be handled in the refactored LLCP as well